common/src/paused_program_storage.rs

```diff
@@ -95,9 +95,17 @@ pub struct ResumeResult<BlockNumber> {
 pub trait PausedProgramStorage: super::ProgramStorage {
     type PausedProgramMap: MapStorage<Key = ProgramId, Value = (Self::BlockNumber, H256)>;
     type CodeStorage: super::CodeStorage;
-    type NonceStorage: ValueStorage<Value = u128>;
-    type ResumeSessions: MapStorage<Key = u64, Value = ResumeSession<Self::BlockNumber>>;
-    type SessionMemoryPages: AppendMapStorage<(GearPage, PageBuf), u64, Vec<(GearPage, PageBuf)>>;
+    type NonceStorage: ValueStorage<Value = SessionId>;
+    type AccountId: Eq + PartialEq;
```

Comment:
not an issue at all but you may define account Id on program storage level to use here Self::AccountId by default